### ISSUE TITLE
Service template for checking interfaces does not exist

### ISSUE THREAD

**Issue by henrikmai** [it-novum/openITCOCKPIT#1446]
After installing the Package for Network devices, I´m unable to add a Switch via the Wizard.

Service template for checking interfaces does not exist
Try to execute `openitcockpit-update` to resolve the issue

I´ve run the command, but nothing changes.
Also reinstalling the Package didn´t solve the problem.

Any Ideas?

**Comment by nook24** [comment]
Happy New Year!
Could you please add the Name of the Wizard you are using and which module you have installed? 
Also did you get any type of error message or so?

**Comment by henrikmai** [comment]
Happy New Year!
The Wizard is **Network**.
The Error Message is ![image](https://user-images.githubusercontent.com/29449777/210210265-83f0cfaa-fcbe-419c-8f2e-3bc225cea018.png)

**Comment by nook24** [comment]
Can you please check if you have installed the package `openitcockpit-module-nwc` and if yes, a Time Period with the id `1` exists on your system?
![grafik](https://user-images.githubusercontent.com/9019992/210213420-a7684e18-41c8-4f95-9ff4-5e4596f95c9a.png)

**Comment by henrikmai** [comment]
I think everything is in place.
![image](https://user-images.githubusercontent.com/29449777/210250714-5c2c9755-2087-44c2-85ce-c1ccba80566c.png)
![image](https://user-images.githubusercontent.com/29449777/210250800-77caa045-76c6-4049-9c0c-1a69755c30a7.png)

**Comment by nook24** [comment]
Hm that's strange. Usually the system will run `openitcockpit-update` automatically, which is responsible for creating missing commands, service templates and so on. Probably that's running into an error.

Could you please check, if the following check commands exists on your system? You should have 11 commands starting with `check_nwc_`

![nwc_commands](https://user-images.githubusercontent.com/9019992/210375778-de5d4b43-2546-4959-b38c-acc45eb885c4.png)

And 15 service templates.
![nwc_servicetemplates](https://user-images.githubusercontent.com/9019992/210375849-aabc6d4f-47d7-4604-a6f7-7ffeac664a09.png)

**Comment by henrikmai** [comment]
Hi,
there are no "check_nwc" commands and no Service templates starting with "NETWORK_NWC" installed.

**Comment by nook24** [comment]
Ok than for sure your system has trouble to update the database. Cloud you please run the command `openitcockpit-update --cc` as `root` user and paste the output to https://gist.github.com/ ?

**Comment by henrikmai** [comment]
https://gist.github.com/henrikmai/41686ae95b510a4c066d659ef6f3668e

**Comment by nook24** [comment]
Errors from the gist: (Just for later reference and people finding this via Google)
```
PHP Fatal error:  Uncaught RuntimeException: Seeds path `/opt/openitc/frontend/plugins/DesignModule/config/Seeds` does not exist and cannot be created because `debug` is disabled. in /opt/openitc/frontend/vendor/cakephp/migrations/src/ConfigurationTrait.php:98 

Importing default records for LinuxBasicMonitoringSSHModule into database
PHP Fatal error:  Uncaught RuntimeException: Migrations path `/opt/openitc/frontend/plugins/LinuxBasicMonitoringSSHModule/config/Migrations` does not exist and cannot be created because `debug` is disabled. in /opt/openitc/frontend/vendor/cakephp/migrations/src/ConfigurationTrait.php:88
```

Oh no. Unfortunately this is caused by a change in the underlying CakePHP framework. 🙁
As a workaround you can run `OITC_DEBUG=1 openitcockpit-update --cc` as `root` user and everything should be working fine.

I will mark this as an bug

**Comment by henrikmai** [comment]
Thanks, that works now!

**Comment by nook24** [comment]
Resolved with openITCOCKPIT 4.5.5: https://github.com/it-novum/openITCOCKPIT/releases/tag/openITCOCKPIT-4.5.5
